inventoryLocalStore/index.js

```diff
@@ -1,10 +1,11 @@
 const { Pool } = require('pg');
 
+const getLengthOfStay = (start, end) =>
+  ((new Date(end) - new Date(start)) / 86400000) + 1;
+
```

Comment:
Might be worth a code explanation for later. Hard to see that 86400000 = 1 day.

Reply:
Noted. I will update PR.